### ISSUE TITLE
Update of `quickcheck-state-machines` broke windows build

### ISSUE THREAD

**Issue by coot** [input-output-hk/ouroboros-network#1753]
The commit which broke windows: https://github.com/advancedtelematic/quickcheck-state-machine/tree/master/src/Test/StateMachine/Lockstep
`Aux` is not a valid path:
```
error: invalid path 'src/Test/StateMachine/Lockstep/Aux.hs'
```

**Comment by mrBliss** [comment]
Waiting for https://github.com/advancedtelematic/quickcheck-state-machine/pull/362 to be merged.

**Comment by mrBliss** [comment]
Should be fixed by #1760.
